### DISCUSSION TITLE
Quarkus support?

### DISCUSSION THREAD

**Discussion by dylanenabled** [ExpediaGroup/graphql-kotlin#1773] [Q&A] [answered]
Hi, Does graphql-kotlin support running on Quarkus? I know that Quarkus has support for smallrye-graphql but graphql-kotlin seems to be based on graphql-java.

**Comment by dariuszkuc** [comment, marked as answer]
Hello 👋 
`graphql-kotlin` comes with built-in support for reactive Spring Boot (autoconfiguration module) and Ktor (plugin) webservers. You can use schema generation module with any other webserver (including Quarkus) but you will have to implement the server logic yourself (i.e. handling of the HTTP requests, wiring up your schema etc).

Thanks,
Derek

---
side note: `smallrye-graphql` (similarly as `graphql-kotlin`) is also an abstraction over `graphql-java`
